pages/api/user/create.ts

```diff
@@ -25,7 +25,7 @@ export default async function create(
       }
     });
 
-    const playerUrl = `${process.env.BASE_URL}${playerData.tournamentId}/users/${user.id}`;
+    const playerUrl = `${process.env.BASE_URL}tournaments/${playerData.tournamentId}/users/${user.id}`;
```

Comment:
```suggestion
    const playerUrl = `${process.env.BASE_URL}/tournaments/${playerData.tournamentId}/users/${user.id}`;
```

Reply:
Jos ei base-url tota sisällä?

Reply:
No, nyt kun saatiin se kauttaviiva sieltä pois niin tehdään näin!